lib/chunks.js

```diff
@@ -988,17 +988,19 @@ module.exports.ensureChunksForCourseAsync = async (courseId, chunks) => {
       // If this chunk is already being loaded, reuse the existing promise
       return pendingChunkPromise;
```

Comment:
We could also add an `await` here for clarity if we want, although it doesn't matter either way.